### PR TITLE
fix: correct sizing of images in replies WPB-1864

### PR DESCRIPTION
**Pull request by tmspzz**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-1864" title="WPB-1864" target="_blank">BugWPB-1864</a>  [replies] replyies to a picture should show the original picture in the right aspect ratio
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [x] contains a reference JIRA issue number like `SQPIT-764`
    - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

When replaying to an image, the image could appear squished depending on the size of the screen and the length of the date line 

### Solutions

Rework the design so that images can stretch vertically

reply-in-conversation
reply-in-message-composer

#### How to Test

Set your locale or zoom level so the left side of a message (text) takes most of the screen. Reply to an image. Notice that the image isn't squished.

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [x] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by alexandreferris — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [169](https://10.10.124.134/job/AR-build-pipeline/169/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.3.0(6942733)(6942733)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.3.0(6942733)(6942733)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [223](https://10.10.124.134/job/AR-build-pipeline/223/) **failed**.

File: app/src/main/kotlin/com/wire/android/ui/home/conversations/messages/QuotedMessage.kt [hunk 1]
```diff
@@ -333,25 +343,104 @@ private fun QuotedImage(
...
+            startContent()
+        }, centerContent = {
+            MainContentText(stringResource(R.string.notification_shared_picture))
+        }, footerContent = {
+            QuotedMessageOriginalDate(originalDateTimeText)
+        })
+
+    } else {
+
+        // Similar to the standard layout, but the space for the mage stretches
```
Comment (mchenani) [inline app/src/main/kotlin/com/wire/android/ui/home/conversations/messages/QuotedMessage.kt]:
```suggestion
        // Similar to the standard layout, but the space for the image stretches
```

Comment (mchenani) [reply app/src/main/kotlin/com/wire/android/ui/home/conversations/messages/QuotedMessage.kt]: :D

**Review by mchenani — APPROVED** [review]
🔥, LGTM but I'm not really a Compose Expert 🦧

**Comment by AndroidBob** [comment]
Build [226](https://10.10.124.134/job/AR-build-pipeline/226/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-v4.3.0(6967113)(6967113)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-v4.3.0(6967113)(6967113)-staging-compat.apk)

**Review by trOnk12 — APPROVED** [review]

**Pull request merged**